tasks/main.yml

```diff
@@ -1,22 +1,27 @@
 ---
 - name: OpenGrok | Install
-  include: install.yml
+  import_tasks: install.yml
   tags:
     - install
```

Comment:
The generic tags like `install` should be removed, shouldn't be?

Reply:
done